ecs-agent/daemonimages/csidriver/driver/node.go

```diff
@@ -273,6 +306,13 @@ func (d *nodeService) NodeUnstageVolume(ctx context.Context, req *csi.NodeUnstag
 		return nil, status.Error(codes.InvalidArgument, "Staging target not provided")
 	}
 
+	// The following code parses the taskID from the given mount path. The target mount path is always in the
+	// format of C:\\ProgramData\\Amazon\\ECS\\ebs\\taskId-volId
```

Comment:
Windows-specific code?

Reply:
Removed in [502174e](https://github.com/aws/amazon-ecs-agent/pull/4326/commits/502174ee6c28610cf856718ec299d900f98eb65b)